legacy/application/forms/GeneralPreferences.php

```diff
@@ -207,6 +207,22 @@ public function init()
         $radioPageLoginButton->setValue($displayRadioPageLoginButtonValue);
         $this->addElement($radioPageLoginButton);
 
+        // add a checkbox for completely disabling the radio page
+        $radioPageDisabled = new Zend_Form_Element_Checkbox('radioPageDisabled');
+        $radioPageDisabled->setDecorators([
+            'ViewHelper',
+            'Errors',
+            'Label',
+        ]);
+        $radioPageDisabledValue = Application_Model_Preference::getRadioPageDisabled();
+        if ($radioPageDisabledValue == '') {
+            $radioPageDisabledValue = false;
+        }
+        $radioPageDisabled->addDecorator('Label', ['class' => 'enable-tunein']);
```

Comment:
Why is this 'enable-tunein' ?

Reply:
This is Copy/Paste from the style of the already existing 'Hide login link' checkbox. I think the previous committer hijacked it somewhere...